plasmapy/physics/quantum.py

```diff
@@ -1,9 +1,21 @@
-import numpy as np
-from astropy import units
-from ..constants import c, h
-from ..atomic import ion_mass
-from ..utils import _check_quantity, _check_relativistic
-from .relativity import Lorentz_factor
+"""
+Functions for quantum parameters, including electron degenerate
+gases and warm dense matter.
+"""
+if __name__ == "__main__":    
```

Comment:
It's not about importing, but rather about running directly. Can you run `python test_quantum.py` directly from the command line when this split is not there? This probably isn't necessary for all modules, but I think it could be valuable for running tests individually.

Reply:
This is just my ignorance of how PyTest works. I'll remove this split and push an update.